module-2/backend/week_two.md

```diff
@@ -5,17 +5,45 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+  A database which can utilize relationships between sets of data.
+  
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+  ActiveRecord Methods
```

Comment:
This question is asking you to give an example of how to use these methods. 

```ruby
class Cat
  belongs_to :owner
end
```

because we have this `belongs_to`, we can call `.owner` on an instance of Cat.